client/app/templates/template-explorer.component.js

```diff
@@ -119,11 +127,25 @@ function ComponentController(ListView, TemplatesService, EventNotifications, $st
   }
 
   function getMenuActions() {
-    const menuActions = [];
+    const menuActions = [{
+      name: __('Edit'),
+      title: __('Edit Template'),
+      actionFn: handleEdit,
+    }];
 
     return menuActions;
   }
+  function listActionDisable(config, items) {
+    const menuCreate = 0;
```

Comment:
These aren't true and false variables, they were just convenience variables so a person could look at the code below it and know what menu element they were editing instead of array indexes.